src/sentry/models/actor.py

```diff
@@ -116,6 +116,10 @@ def from_actor_identifier(cls, actor_identifier: Union[int, str]) -> "ActorTuple
             "maiseythedog" -> look up User by username
             "maisey@dogsrule.com" -> look up User by primary email
         """
+
+        if actor_identifier is None:
+            return None
```

Comment:
Should we make a docstring update?
Should we add a comment here as to why may we end up here with no identifier?